app/services/paypal_service/logger.rb

```diff
@@ -2,29 +2,28 @@ module PaypalService
   class Logger
 
     def info(msg)
-      #TODO: Fix logging totally
-      Rails.logger.warn "[Paypal] #{msg}"
+      Rails.logger.info ({ paypal: msg }.to_json)
```

Comment:
Maybe this could be renamed to something else, something more general, like `log_json`, `log_paypal_message` e.g.